docs/notebooks/content/Text-Summerization-gpt.ipynb

```diff
@@ -0,0 +1,208 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "4vXfYHX6QSJu"
+      },
+      "source": [
+        "# Generate Synthetic text summarization with Gretel GPT\n",
+        "\n",
+        "* In this notebook we use Gretel GPT with Llama-2 7b model to create synthetic text summerization dataset.  \n",
+        "* To run this notebook, you will need an API key from the [Gretel Console](https://console.gretel.ai/)."
```

Comment:
question: Is this the same link we use in other notebooks? Might be nice to a link here directly to the API key page, so something like https://console.gretel.ai/users/me/key